wordstats/edit_distance_function_factory.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABC, abstractmethod
+from wordstats.cognate_files_path import *
+import configparser
+
+
+# abstract class for creating a distance measure between two rules
+# support for loading from .cfg file and loading rules is supplied
+# abstract methods are specific to the distance measure
+class WordDistanceFactory(ABC):
+
+    def __init__(self, primary_language, secondary_language):
+        super().__init__()
+        self.rules = []
+        self.threshold = 0.5
+    # in addition, derived classes need to implement the following variables
+        self.method_name = ""
+        self.languageFrom = primary_language
+        self.languageTo = secondary_language
```

Comment:
here we talk about languageFrom and languageTo. 
in the other class we had primary and secondary.
we should choose *one* convention, and use it all throughout the project.